source/frontends/libretro/libretro.cpp

```diff
@@ -353,6 +358,24 @@ bool retro_load_game(const retro_game_info *info)
             std::swap(ourGame, game);
         }
 
+        // capture initial memory state
+        ourGame->flushAndCaptureMemory();
+
+        // define memory map to allow exposing both main RAM and aux RAM.
+        // retro_get_memory_data will continue to return just main RAM.
+        struct retro_memory_descriptor descs[2];
+        struct retro_memory_map        mmaps;
+        memset(descs, 0, sizeof(descs));
+        descs[0].ptr = MemGetBankPtr(0, true);
+        descs[0].start = 0x0000;
```

Comment:
not a big deal, but use `_6502_MEM_BEGIN` and `_6502_MEM_LEN`

Reply:
done